source/localizable/tutorial/autocomplete-component.md

```diff
@@ -17,7 +17,7 @@ As before when we created the [`rental-listing` component](../simple-component),
 In our `app/templates/rentals.hbs` template file, we'll add a reference to our new `list-filter` component.
 
 Notice that below we "wrap" our rentals markup inside the open and closing mentions of `list-filter` on lines 12 and 20.
-This is an example of the [**block form**](/components/wrapping-content-in-a-component) of a component,
+This is an example of the [**block form**](/v2.12.0/components/wrapping-content-in-a-component) of a component,
```

Comment:
Hi, this isn't quite right, because we'd need to update the links for every version!
It needs to be relative, so `../components` like in https://github.com/emberjs/guides/pull/1910/files#diff-e0faf55781ad58791e0623fbe42fe37dL2. Can you make that tweak, please?